src/PsrLoggerAbstractAdapterFactory.php

```diff
@@ -5,13 +5,18 @@
 namespace Laminas\Log;
 
 use Interop\Container\ContainerInterface;
+use Interop\Container\Exception\ContainerException;
+use Laminas\ServiceManager\AbstractFactoryInterface;
+use Laminas\ServiceManager\ServiceLocatorInterface;
+use Psr\Container\ContainerExceptionInterface;
+use Psr\Container\NotFoundExceptionInterface;
 
 /**
  * PSR Logger abstract service factory.
  *
  * Allow to configure multiple loggers for application.
  */
-class PsrLoggerAbstractAdapterFactory extends LoggerAbstractServiceFactory
+class PsrLoggerAbstractAdapterFactory implements AbstractFactoryInterface
```

Comment:
Potentially minor detail, but `protected processConfig()` no longer exists on this symbol now.

Anyway, OK with glancing over it